avocado/utils/lv_utils.py

```diff
@@ -335,7 +335,7 @@ def vg_create(vg_name, pv_list, force=False):
         pv_list = " ".join(pv_list)
     else:
         pv_list = str(pv_list)
-    cmd += " %s %s" % (vg_name, pv_list)
+    cmd += " %s %s -y" % (vg_name, pv_list)
     process.run(cmd, sudo=True)
```

Comment:
@narasimhan-v wouldn't it make more sense if the the `-y` flag was given only if the `force` parameter was set? They seem to be very much correlated:

```
       -f|--force ...
              Override various checks, confirmations and protections.  Use with extreme caution.
...
       -y|--yes
              Do not prompt for confirmation interactively but always assume the answer yes. Use with extreme caution (For automatic no, see -qq.)
```

Reply:
@narasimhan-v since you think it's a good idea, please send an updated version and I'll merge it ASAP. Thanks!

Reply:
On it.

Reply:
@clebergnu Done, thanks !